modyn/config/schema/sampling/downsampling_config.py

```diff
@@ -99,6 +99,15 @@ class NoDownsamplingConfig(BaseDownsamplingConfig):
     ratio: Literal[100] = 100
 
 
+class ILTrainingConfig(ModynBaseModel):
+    num_workers: int = Field("Number of workers to use for training.", min=1)
+    il_model_id: str = Field("The model class name to use as the IL model.")
+    il_model_config: dict = Field(
+        default_factory=dict, description="Configuration dictionary that will be passed to the model on initialization."
+    )
+    amp: bool = Field(False, description="Whether to use automatic mixed precision.")
+
```

Comment:
It's not a full list of needed training configs but that's what needed to finish this step.